src/main/java/core/basesyntax/GetArea.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface GetArea {
```

Comment:
This is not a proper name for an interface - make sure that you've read the guidelines before posting a solution: https://github.com/mate-academy/jv-oop-advanced/blob/master/checklist.md
![obraz](https://github.com/mate-academy/jv-oop-advanced/assets/19765018/688ada62-d888-4974-b8c9-705816dbd088)